app/hooks/useLazyMinted.tsx

```
@@ -0,0 +1,50 @@
+import { videoContract } from '@app/lib/sdk/thirdweb/get-contract';
+import { NFT } from '@app/types/nft';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { getNFTs } from 'thirdweb/extensions/erc1155';
+import { useActiveAccount } from 'thirdweb/react';
+
+export default function useLazyMinted() {
+  const activeAccount = useActiveAccount();
+
+  const [nfts, setNFTs] = useState<NFT[]>([]);
+  const [isProcessing, setIsProcessing] = useState(false);
+  const [error, setError] = useState<any>({});
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)